torchx/test/torchx/random_test.exs

```diff
@@ -65,7 +65,7 @@ defmodule Torchx.Nx.RandomTest do
       # Output does not match Nx because of the sign of the remainder.
       distribution_case(:randint_split,
         args: [0, 10, [shape: {5}]],
-        expected: Nx.tensor([3, 2, 6, 0, 0], type: :s64)
+        expected: Nx.tensor([1, 1, 4, 1, 9], type: :s64)
```

Comment:
I didn't really understand why these values changed, but that's ok.

Reply:
It's because in `args: [0, 10, [shape: {5}]]` passed to defn the numbers `0` and `10` are now passed as s32 rather than s64. We could maintain the behaviour with ``args: [Nx.s64(0), Nx.s64(10), [shape: {5}]]``, but changing the test is equally fine.